modules/internal/DefaultRectangular.chpl

```diff
@@ -628,7 +726,21 @@ module DefaultRectangular {
           yield dsiAccess(i);
       }
     }
-  
+
+    iter these(param tag: iterKind,
+               tasksPerLocale = dataParTasksPerLocale,
+               ignoreRunning = dataParIgnoreRunningTasks,
+               minIndicesPerTask = dataParMinGranularity)
+      ref where tag == iterKind.standalone {
```

Comment:
I'm not sure about this, but does this where clause also need to check for sublocales?